doc/developers/kv/api.rst

```diff
@@ -1,12 +1,16 @@
 Key-Value Store API
 ===================
 
+This page presents the API that a CCF application must use to read and mutate the replicated key-value store.
+
+A CCF application should define one or multiple public or private :cpp:class:`kv::Map`. Then, each :cpp:class:`ccf::HandlerRegistry::Handler` should use the :cpp:class:`kv::Tx` transaction object to read and write to specific :cpp:class:`kv::Map`.
+
 Store
 -----
 
 .. doxygenclass:: kv::Store
    :project: CCF
-   :members:
+   :members: create, get
```

Comment:
Only `create()` and `get()` gets documented. Obviously not the long-term solution but might prevent some users from calling other methods.